Numeric/Units/Dimensional/DK/Dimensions/TermLevel.hs

```diff
@@ -0,0 +1,67 @@
+{-# OPTIONS_HADDOCK not-home, show-extensions #-}
+module Numeric.Units.Dimensional.DK.Dimensions.TermLevel
+(
+  Dimension'(..),
+  HasDimension(..),
+  dOne,
+  dLength, dMass, dTime, dElectricCurrent, dThermodynamicTemperature, dAmountOfSubstance, dLuminousIntensity,
+  (*), (/), (^), recip,
+  asList
+)
+where
+
+import Prelude hiding ((*), (/), (^), recip)
+import qualified Prelude as P
+
+-- | A physical dimension, encoded as 7 integers, representing a factorization of the dimension into the
+-- 7 SI base dimensions. By convention they are stored in the same order as 
+-- in the 'Numeric.Units.Dimensional.DK.Dimensions.TypeLevel.Dimension' data kind.
+data Dimension' = Dim' !Int !Int !Int !Int !Int !Int !Int 
+  deriving (Show, Eq, Ord)
+
+-- | The monoid of dimensions under multiplication.
+instance Monoid Dimension' where
```

Comment:
I need to add `import Data.Monoid (Monoid(..))` for this to build. Another base-4.6 thing.